test/e2e/mesos.go

```diff
@@ -87,7 +87,7 @@ var _ = Describe("Mesos", func() {
 			ObjectMeta: api.ObjectMeta{
 				Name: podName,
 				Labels: map[string]string{
-					"k8s.mesosphere.io/roles": "role1",
+					"k8s.mesosphere.io/roles": "public",
 				},
 			},
 			Spec: api.PodSpec{
```

Comment:
At least I'd leave non-conflicting ports (7000-7999) so we can schedule nginx-like pods in the dev environment.